Editor/Window/AnywherePage.cs

```diff
@@ -54,13 +60,20 @@ private void SetupStatusBoxes()
 
         private void UpdateGui()
         {
+            _launchClientFoldout.AddToClassList(InactiveFoldoutElementName);
```

Comment:
There seems to have been a misunderstanding. We should not be hiding the foldout. We should **disable** the button in the foldout until the current profile is ready to run (bootstrapped, has a fleet, has a registered compute)

Reply:
Looks like https://github.com/aws/amazon-gamelift-plugin-unity/pull/192 already handles the enable/disable for the button. I'm going to close out this PR in favor of 192